code/__DEFINES/atmospherics.dm

```diff
@@ -90,8 +90,8 @@
 #define OPEN_HEAT_TRANSFER_COEFFICIENT		0.4
 /// a hack for now
 #define WINDOW_HEAT_TRANSFER_COEFFICIENT	0.1
-/// a hack to help make vacuums "cold", sacrificing realism for gameplay
-#define HEAT_CAPACITY_VACUUM				2000
+/// space is much more realistic now, still cold as fuck but that can be "radiation"
```

Comment:
Comments don't need to explain what code used to be. Explain what it is now, especially in a doc comment.